topi/python/topi/arm_cpu/conv2d.py

```diff
@@ -810,25 +954,19 @@ def _conv2d_legalize(attrs, inputs, arg_types):
     if attrs['data_layout'] == 'NHWC':
         data, kernel = inputs
         if attrs['kernel_layout'] == 'HWIO':
-            # Handle HWIO layout. This is common in TF graph.
-            kernel = relay.transpose(kernel, axes=(3, 2, 0, 1))
+            # HWIO layout is expected for NHWC input.
```

Comment:
the assumption here is that we run `bitserial_conv2d_legalize` beforehand?

Reply:
reverted changes to arm nhwc schedule in favor of jackwish version.